fxa/tests/__init__.py

```diff
@@ -1,3 +1,7 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from requests.packages import urllib3
+urllib3.disable_warnings()
```

Comment:
Even if it was disabled in production it should not be a problem since it is only a warning related to the Python version, since it is used in all test, I wouldn't add this in every testcase. Should I create a BaseTestCase and inherit every FxA TestCase with it?